source/app/store/entities/documents/actions.js

```diff
@@ -182,12 +183,15 @@ export const fetchDocument = createAction(FETCH_DOCUMENT, async documentid => {
   );
 
 // Get the raw comprehend medical response data from a json file on S3
- const s3ComprehendMedicalResponse = await Storage.get(comprehendMedicalResponsePath, {
-  download: true
-});
-const comprehendMedicalRespone = JSON.parse(
-  s3ComprehendMedicalResponse.Body ? s3ComprehendMedicalResponse.Body.toString() : null
-);
+let comprehendMedicalRespone = null;
```

Comment:
Are you sure that the `null` propagating throughout the app wouldn't break anything? It was previously always an object.

Reply:
Yep I tested it and it does not break anything. In fact https://github.com/awslabs/document-understanding-solution/blob/dcf2904d3f7c272659c48d9697090b90bc9fe793/source/app/store/entities/documents/actions.js#L188 this line anyways assigns it to null if the response does not have any body.